spec/swagger_helper.rb

```diff
@@ -698,6 +698,74 @@
               amount: { "$ref" => "#/components/schemas/currency" },
             },
           },
+          SelfEmploymentType: {
+            type: :string,
+            enum: %i[partnership company_director sole_trader],
```

Comment:
This misses off LLP. There may be other business structures too.

I wonder if we can simplify this by just accepting two clearer answers?:
* 'the client is responsible for the debts of the business' - ie sole trader OR partner in a (normal) partnership
* 'the business is a separate entity from the client' - ie company director of limited company OR partner of a LLP

Reply:
@davidread not according to the Miro - all different types (apparently) have different calculations

Reply:
ok makes sense, let's keep them like this then